src/BlockTypes/AbstractBlock.php

```diff
@@ -76,14 +76,16 @@ public function __construct( AssetApi $asset_api, AssetDataRegistry $asset_data_
 	 *
 	 * @param array|WP_Block $attributes Block attributes, or an instance of a WP_Block. Defaults to an empty array.
 	 * @param string         $content    Block content. Default empty string.
+	 * @param WP_Block       $block      Block instance.
 	 * @return string Rendered block type output.
 	 */
-	public function render_callback( $attributes = [], $content = '' ) {
+	public function render_callback( $attributes = [], $content = '', $block ) {
```

Comment:
On PHP 8 this line is giving me the following error:

`( ! ) Deprecated: Required parameter $block follows optional parameter $attributes in [snip]/app/public/wp-content/plugins/woocommerce-blocks/src/BlockTypes/AbstractBlock.php on line 82`